File: FastScript/Demos/Main/Samples/BasicScript/file.vb
' FastScript v1.0          
' Files demo               


dim fs, s

  fs = new TFileStream("test.txt", fmCreate)

  s = "Testing file..."
  fs.Write(s, Length(s))
  fs.Write(#13#10, 2)
  s = "Tested OK!"#13#10
  fs.Write(s, Length(s))

  delete fs

File: FastScript/Demos/Main/Samples/C++Script/file.cpp
/**************************}
{ FastScript v1.0          }
{ Files demo               }
{**************************/

{
  TFileStream fs = new TFileStream("test.txt", fmCreate);

  String s = "Testing file...";
  fs.Write(s, Length(s));
  fs.Write("\r\n", 2);
  s = "Tested OK!\r\n";
  fs.Write(s, Length(s));

  delete fs;
}